src/components/views/passwordReset/NewPassword.js

```diff
@@ -12,17 +13,35 @@ const ResetPassword = ({
   const [formData, setFormData] = useState({
     password: '',
     confirmPassword: '',
+    error: '',
   });
 
-  const { password, confirmPassword } = formData;
-  const onChange = event => setFormData({ ...formData, [event.target.name]: event.target.value });
+  const { password, confirmPassword, error } = formData;
+
+  const passwordValidation = (event) => {
+    event.persist();
+    const status = validatePassword(event.target.value);
+    setFormData(prevState => ({ ...prevState, error: !status ? 'Password must contain at least one uppercase letter, one lowercase letter and one numeric digit' : '' }));
+    setFormData(prevState => ({ ...prevState, [event.target.name]: event.target.value }));
+    return status;
+  };
+
+  const confirmPassworValidation = (event) => {
+    event.persist();
+    setFormData(prevState => ({ ...prevState, [event.target.name]: event.target.value }));
+  };
+
+  useEffect(() => {
+    const isMatch = passwordMatch(formData);
+    setFormData(prevState => ({ ...prevState, error: !isMatch ? 'Password does not match' : '' }));
+  }, [confirmPassword]);
 
   const handleSubmit = (event) => {
     event.preventDefault();
     resetPassword(password, confirmPassword, params.token);
   };
 
-  const error = !success && message;
+  const err = !success && message;
```

Comment:
Why will you change a proper variable name to this??

Reply:
I have another error variable on line 19.

Reply:
You added that in this PR. Instead of changing what you met on the file, why not give your variable a different name?

Reply:
Makes sense... I'm returning the old name now.